template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/_LoginPartial.cshtml

```diff
@@ -0,0 +1,68 @@
+@*#if (IndividualB2CAuth)
+@using Microsoft.AspNetCore.Authentication.Extensions
+@using Microsoft.Extensions.Options
+
+#endif
```

Comment:
I think this should be
`#endif *@`

Or possibly get rid of this #endif and the next line since both the @*#if on line 1 and on line 6 are conditioned on the same thing.